R/caching.R

```diff
@@ -168,10 +181,13 @@ rec_checkout_repos <- function(repos_to_process, feature, direction = c("upstrea
   hashed_repos_to_process <- vapply(repos_to_process, hash_repo_and_host, character(1))
   rm(repos_to_process)
 
-  # only one of them may be filled depending on the direction
-  # should not be c() since assigned to list below removes element otherwise
-  upstream_deps_graph <- list()
-  downstream_deps_graph <- list()
+  hashed_processed_repos <- character(0)
```

Comment:
update function doc, what exactly is returned, distincttion btwenn dependency graph from yaml and description file

Reply:
I find this function confusing and would revert it to its old state. Then get the package dependencies (according to DESCRIPTKION) in another loop.

Reply:
no longer relevant after refactor